src/components/admin/CategoryAdminCard.tsx

```diff
@@ -115,6 +114,39 @@ const UPDATE_CATEGORY_POSITION = gql`
 `
 const DELETE_CATEGORY = gql`
   mutation DELETE_CATEGORY($categoryId: String!) {
```

Comment:
member_task can use this to delete relationship
```
  update_member_task(where: {category_id: {_eq: ""}}, _set: {category_id: null}) {
    returning {
      id
    }
  }
```

Reply:
had modified